.travis.yml

```diff
@@ -4,6 +4,9 @@
 language: objective-c
 
 env:
+  matrix:
+    
+    - CONDA_PERL=5.20.3.1
```

Comment:
I don't really see how we get around either setting this environment variable or passing it as an argument given that it takes a [hardcoded default value]( https://github.com/conda/conda-build/blob/cae2e6ccaf8931acd72c40098027fa698ab41030/conda_build/config.py#L22 ).